components.js

```diff
@@ -578,6 +578,10 @@ var components = {
 			"require": "markup",
 			"owner": "Golmote"
 		},
+		"tsx": {
+			"title": "React Tsx", 
+			"require": ["markup", "javascript", "jsx", "typescript"]
```

Comment:
I think there is no need to require `markup` and `javascript` here, since they are already requirements for `jsx`.

Reply:
Also, could you please move the component up so that the alphabetical order is based on the `title` value instead of the key. I know this is kinda counter-intuitive, but it's the way we did with JSX, so let's keep it consistent.